src/components/editor/ResourceComponent.jsx

```diff
@@ -51,7 +51,7 @@ const ResourceComponent = () => {
           <CopyToNewMessage />
           <div className="row">
             <div className="col-md-11">
-              <ResourceURIMessage />
+              <ResourceURIMessage resourceKey={resourceKey} />
             </div>
```

Comment:
this component was edited to take this parameter so it can be re-used by the new preview header component I created